storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6975,15 +6973,13 @@ static void xtrabackup_prepare_func(int argc, char **argv) {
   /* Check whether the log is applied enough or not. */
   if ((xtrabackup_incremental && log_get_lsn(*log_sys) < incremental_to_lsn) ||
       (!xtrabackup_incremental && log_get_lsn(*log_sys) < metadata_to_lsn)) {
-    msg("xtrabackup: error: "
-        "The transaction log file is corrupted.\n"
-        "xtrabackup: error: "
-        "The log was not applied to the intended LSN!\n");
-    msg("xtrabackup: Log applied to lsn " LSN_PF "\n", log_get_lsn(*log_sys));
+    xb::error() << "The transaction log file is corrupted.";
+    xb::error() << "The log was not applied to the intended LSN!";
+    xb::info() << "Log applied to lsn " << log_get_lsn(*log_sys);
     if (xtrabackup_incremental) {
-      msg("xtrabackup: The intended lsn is " LSN_PF "\n", incremental_to_lsn);
+      xb::info() << "The intended lsn is " << incremental_to_lsn;
```

Comment:
```suggestion
      xb::error() << "The intended lsn is " << incremental_to_lsn;
```

we are continuing to print the error from before if branch

Reply:
good catch